src/transaction/log_tran_table.c

```diff
@@ -1695,7 +1694,6 @@ logtb_initialize_tdes (LOG_TDES * tdes, int tran_index)
   LSA_SET_NULL (&tdes->topop_lsa);
   LSA_SET_NULL (&tdes->tail_topresult_lsa);
   LSA_SET_NULL (&tdes->commit_abort_lsa);
-  tdes->last_mvcc_lsa.set_null ();
```

Comment:
Is mvccinfo.reset() not needed here?

Reply:
`logtb_initialize_tdes` already calls `mvccinfo.init` ().
So removing this is ok.